opennms-doc/guide-install/src/asciidoc/text/newts/introduction.adoc

```diff
@@ -0,0 +1,21 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[gi-install-ts-newts]]
+== Installing Time Series database Newts
+
+link:http://newts.io/[Newts] is a time-series data store based on link:http://cassandra.apache.org/[Apache Cassandra].
+_Newts_ is a persistence strategy, that can be used as an alternative to link:http://www.opennms.org/wiki/JRobin[JRobin] or link:http://oss.oetiker.ch/rrdtool/[RRDtool].
+
+IMPORTANT: It is currently not supported to initialize the _Newts_ keyspace from _Microsoft Windows Server_ operating system.
+           _Microsoft Windows_ based _Cassandra_ server can be part of the cluster, the initializing can just be executed from a _Linux_ system.
```

Comment:
...cluster, but keyspace initialization is only possible using a _Linux_-based system.